docs/install/upgrading.md

```diff
@@ -17,34 +17,7 @@ $ infra version
 
 ## Upgrading Infra CLI
 
-{% tabs %}
-{% tab label="macOS" %}
-```
-brew update
-brew upgrade infra
-```
-{% /tab %}
-{% tab label="Windows" %}
-```powershell
-scoop update infra
-```
-{% /tab %}
-{% tab label="Linux" %}
-
-#### Ubuntu & Debian
-
-```
-sudo apt update
-sudo apt upgrade infra
-```
-
-#### Fedora & Red Hat Enterprise Linux
-
-```
-sudo dnf update infra
-```
-{% /tab %}
-{% /tabs %}
+{% partial file="../partials/cli-install.md" /%}
```

Comment:
Why remove this if the commands are slightly different?

Reply:
Because I didn't read it correctly and assumed it was the same. Will revert